site/src/docs/components/footer/index.mdx

```diff
@@ -78,32 +78,32 @@ The default footer uses the black text colour variant and includes all three sec
 <PlaygroundPreview>
   <Footer title="Helsinki Design System" footerAriaLabel="HDS Footer">
     <Footer.Navigation navigationAriaLabel="HDS Footer navigation">
-      <Footer.Item label="Item" />
-      <Footer.Item label="Item" />
-      <Footer.Item label="Item" />
-      <Footer.Item label="Item" />
-      <Footer.Item label="Item" />
-      <Footer.Item label="Item" />
-      <Footer.Item label="Item" />
-      <Footer.Item label="Item" />
+      <Footer.NavigationLink label="Item" />
+      <Footer.NavigationLink label="Item" />
+      <Footer.NavigationLink label="Item" />
+      <Footer.NavigationLink label="Item" />
+      <Footer.NavigationLink label="Item" />
+      <Footer.NavigationLink label="Item" />
+      <Footer.NavigationLink label="Item" />
+      <Footer.NavigationLink label="Item" />
     </Footer.Navigation>
     <Footer.Utilities backToTopLabel="Back to top">
       <Footer.SoMe>
-        <Footer.Item icon={<IconFacebook />} />
-        <Footer.Item icon={<IconTwitter />} />
-        <Footer.Item icon={<IconInstagram />} />
-        <Footer.Item icon={<IconYoutube />} />
-        <Footer.Item icon={<IconTiktok />} />
+        <Footer.NavigationLink icon={<IconFacebook />} />
+        <Footer.NavigationLink icon={<IconTwitter />} />
+        <Footer.NavigationLink icon={<IconInstagram />} />
+        <Footer.NavigationLink icon={<IconYoutube />} />
+        <Footer.NavigationLink icon={<IconTiktok />} />
       </Footer.SoMe>
-      <Footer.Item label="Contact us" />
-      <Footer.Item label="Give feedback" />
+      <Footer.NavigationLink label="Contact us" />
+      <Footer.NavigationLink label="Give feedback" />
     </Footer.Utilities>
     <Footer.Base copyrightHolder="Copyright" copyrightText="All rights reserved">
-      <Footer.Item label="Link" />
-      <Footer.Item label="Link" />
-      <Footer.Item label="Link" />
-      <Footer.Item label="Link" />
-      <Footer.Item label="Link" />
+      <Footer.NavigationLink label="Link" />
+      <Footer.NavigationLink label="Link" />
+      <Footer.NavigationLink label="Link" />
+      <Footer.NavigationLink label="Link" />
+      <Footer.NavigationLink label="Link" />
```

Comment:
Base links that are smaller in font size should have an XS-sized icon if the link is an external link.